install.sh

```diff
@@ -447,6 +453,15 @@ do_install() {
 			exit 0
 			;;
 		centos|fedora|rhel)
+		
+		   	arch=$(uname -m)
+		    	if [ $arch != "s390x" ] && [ $lsb_dist == "rhel" ]; then
+				echo "Docker-CE is not supported on RHEL for $arch!!!"
+				exit 1
+			else
+				echo "Installing Docker-CE on $arch!"
+			fi
+		   
 			yum_repo="$DOWNLOAD_URL/linux/$lsb_dist/$REPO_FILE"
```

Comment:
Looks like there's some "bashisms" in here (`==` instead of `=`), some quoting missing, and I saw some mixed tab/spaces for indentation.

The "Installing Docker-CE" looks a bit too verbose (I don't think we do so for other installs). Perhaps simplify to:

```suggestion
			if [ "$(uname -m)" != "s390x" ] && [ "$lsb_dist" = "rhel" ]; then
				echo "Docker-CE currently only supported on RHEL for s390x"
				exit 1
			fi
			yum_repo="$DOWNLOAD_URL/linux/$lsb_dist/$REPO_FILE"
```

Reply:
Updated the script with mentioned changes.